WeakAurasOptions/ActionOptions.lua

```diff
@@ -229,19 +229,49 @@ function WeakAuras.AddActionOption(id, data)
         values = WeakAuras.glow_action_types,
         disabled = function() return not data.actions.start.do_glow end
       },
+      start_glow_frame_type = {
+        type = "select",
+        width = WeakAuras.normalWidth,
+        desc = function()
+          return (
+            data.actions.start.glow_frame_type == "UNITFRAME"
+            or data.actions.start.glow_frame_type == "NAMEPLATE"
+          )
+          and L["Require unit from trigger"] or nil
+        end,
+        name = L["Glow Frame Type"],
+        order = 10.3,
+        values = {
+          UNITFRAME = L["Unit Frame"],
+          NAMEPLATE = L["Name Plate"],
+          FRAMESELECTOR = L["Frame Selector"]
+        },
+        hidden = function() return not data.actions.start.do_glow end
+      },
+      start_glow_type = {
+        type = "select",
+        width = WeakAuras.normalWidth,
+        name = L["Glow Type"],
+        order = 10.4,
+        values = WeakAuras.glow_types,
+        hidden = function() return not data.actions.start.do_glow end,
```

Comment:
The glow type should not be visible if the glow action is "Hide".

Same for the OnHide action.